.gitignore

```diff
@@ -24,3 +24,4 @@ Session.vim
 
 # Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
 .idea/
+*.iml
```

Comment:
Also, can you drop this change? This is requiring root approval

Reply:
done